.github/PULL_REQUEST_TEMPLATE/docs.md

```diff
@@ -0,0 +1,37 @@
+## Description
+
+Closes: #XXXX
+
+<!-- Add a description of the changes that this PR introduces and the files that
+are the most critical to review. -->
+
+<!-- Pull requests that sit inactive for longer than 30 days will be closed.  -->
+---
+
+### Author Checklist
+
+*All items are required. Please add a note to the item if the item is not applicable and
+please add links to any relevant follow up issues.*
+
+I have...
+
+- [ ] included the correct `docs:` prefix in the PR title
+- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
+- [ ] provided a link to the relevant issue or specification
+- [ ] reviewed "Files changed" and left comments if necessary
+- [ ] confirmed all CI checks have passed
```

Comment:
also need to check docs and version are matching

Reply:
What do you mean?

Reply:
Not sure if this is what @yaruwangway meant but at minimum if a PR includes a version bump of a dependency, there should be a check that it's compatible with other potential dependencies listed in `go.mod`

Reply:
@glnro `docs.md` should just be used for PRs that change only the docs. There should be no version bumps.